src/lms-ds-loader/tests_integration_sql/conftest.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Tuple
+import pytest
+from unittest.mock import MagicMock
+from pandas import DataFrame
+from sqlalchemy import create_engine
+from sqlalchemy.engine.base import Engine, Connection, Transaction
+
+from edfi_lms_ds_loader.migrator import migrate
+from edfi_lms_ds_loader.mssql_lms_operations import MssqlLmsOperations
+
+
+def _new_mssql_engine() -> Engine:
+    return create_engine(
+        "mssql+pyodbc://localhost,1433/test_integration_lms_toolkit?driver=ODBC+Driver+17+for+SQL+Server?Trusted_Connection=yes",
```

Comment:
I'm not worried about it right now, but we should probably find a way to pass in a different server or at least port number, for example if someone wants to use SQL Server only in a container and has it mapped to a different port. Also would need to configure password authentication in that case.